From 105877850747e60b4b4ebec4308e6a3d27d2db63 Mon Sep 17 00:00:00 2001
From: Sandy Huang <hjc@rock-chips.com>
Date: Tue, 23 Jun 2020 11:27:37 +0800
Subject: [PATCH] drm/rockchip: remove repeated function calls

drm_vblank_cleanup will be called as following path, so delete it here:
drm_dev_put(drm_dev);
    ->drm_dev_release
        ->drm_dev_fini(dev);
            -> drm_vblank_cleanup(dev);

Change-Id: Ie7d31c19f00cdf8bb331bf069ce09046e6fe6ea4
Signed-off-by: Sandy Huang <hjc@rock-chips.com>
---
 drivers/gpu/drm/rockchip/rockchip_drm_drv.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/gpu/drm/rockchip/rockchip_drm_drv.c b/drivers/gpu/drm/rockchip/rockchip_drm_drv.c
index eb3919ccdbb1..e723ef9dabca 100644
--- a/drivers/gpu/drm/rockchip/rockchip_drm_drv.c
+++ b/drivers/gpu/drm/rockchip/rockchip_drm_drv.c
@@ -1551,7 +1551,6 @@ static int rockchip_drm_bind(struct device *dev)
 err_kms_helper_poll_fini:
 	rockchip_gem_pool_destroy(drm_dev);
 	drm_kms_helper_poll_fini(drm_dev);
-	drm_vblank_cleanup(drm_dev);
 err_unbind_all:
 	component_unbind_all(dev, drm_dev);
 err_mode_config_cleanup:
@@ -1574,7 +1573,6 @@ static void rockchip_drm_unbind(struct device *dev)
 	rockchip_gem_pool_destroy(drm_dev);
 	drm_kms_helper_poll_fini(drm_dev);
 
-	drm_vblank_cleanup(drm_dev);
 	drm_atomic_helper_shutdown(drm_dev);
 	component_unbind_all(dev, drm_dev);
 	drm_mode_config_cleanup(drm_dev);
-- 
2.35.3

